src/test/java/com/google/sps/servlets/LiveCourseDataTest.java

```diff
@@ -123,16 +141,31 @@ private void createRequest(
     return result;
   }
 
-  private void addRatingEntity(DatastoreService db, int hours, int difficulty, Key parent) {
+  private void addRatingEntity(
```

Comment:
Since this method has so many params and they are often the same type it's good practice to comment what the params are when the method is called (you can see other examples of this above).